cli/lockfile.rs

```diff
@@ -123,6 +184,63 @@ impl Lockfile {
     let checksum = crate::checksum::gen(&[code.as_bytes()]);
     self.content.remote.insert(specifier.to_string(), checksum);
   }
+
+  /// Checks the given module is included.
+  /// Returns Ok(true) if check passed.
+  fn check_npm_package(&mut self, package: &NpmResolutionPackage) -> bool {
+    let specifier = package.id.serialize_for_lock_file();
+    if let Some(package_info) = self.content.npm.packages.get(&specifier) {
+      // TODO(bartlomieju): remove this unwrap
+      package_info.integrity == package.dist.integrity.clone().unwrap()
```

Comment:
@dsherret any particular reason that `integrity` is `Option<String>` instead of `String`?

Reply:
Shouldn't we default to using `shasum` then?

Reply:
I guess we should use `integrity` if it's available and fallback to `shasum`, at least that appears to be what [`package-lock.json`](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#packages) does.